src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/admin/AdminPageIT.java

```diff
@@ -24,7 +24,12 @@ public void accessPageAsAdmin() {
 		AdminPage page = AdminPage.initPage(driver());
 		assertTrue("Admin button should be displayed", page.adminPanelButtonVisible());
 		page.clickAdminButton();
```

Comment:
Why are 2/3 tests on this page ignored?  Should this PR not be getting them all functional?